lib/browser/api/electron_api_power_monitor_win.cc

```diff
@@ -0,0 +1,44 @@
+#include "lib/browser/api/electron_api_power_monitor_win.h"
```

Comment:
What's the purpose of this file? Is it called anywhere?

I'd expect changes to go into `shell/browser/api/electron_api_power_monitor_win.cc`.